drivers/can/can_stm32.c

```diff
@@ -856,7 +918,8 @@ static void config_can_1_irq(CAN_TypeDef *can)
 		    can_stm32_tx_isr, DEVICE_GET(can_stm32_1), 0);
 	irq_enable(DT_CAN_1_IRQ_SCE);
 #endif
-	can->IER |= CAN_IT_TME | CAN_IT_ERR | CAN_IT_FMP0 | CAN_IT_FMP1;
+	can->IER |= CAN_IER_TMEIE | CAN_IER_ERRIE | CAN_IER_FMPIE0 |
```

Comment:
Is that stritcly HAL related?
If not, can you move it to a separate commit?

Reply:
It is. CAN_IT_TME is from HAL and CAN_IER_TMEIE from mcu headers.